unstructured/ingest/logger.py

```diff
@@ -14,3 +14,14 @@ def ingest_log_streaming_init(level: int) -> None:
         logger.addHandler(handler)
 
     logger.setLevel(level)
+
+
+def make_default_logger(level: int) -> logging.Logger:
```

Comment:
what is the idea here? why is notion the only one using this?

Reply:
I found this to be the better pattern but to limit how much is changes in the code base only the new notion code is using this.